README.md

```diff
@@ -125,6 +129,77 @@ Kafka. This value cannot be set, only read.
 
 Aiven ID format when importing existing resource: name of the project as is.
 
+### Resource Account
+```
+resource "aiven_account" "account1" {
+    name = "<ACCOUNT_NAME>"
+}
+
+```
+
+`name` defines an account name.
+
+`account_id` is an auto-generated unique account id.
+
+### Resource Account Team
+```
+resource "aiven_account_team" "account_team1" {
+    account_id = aiven_account.<ACCOUNT_RESOURCE>.account_id
+    name = "account_team1"
+    
+    # invite users to a team
+    invite_users = "user+1@example.com,user+2@example.com"
```

Comment:
This invite_users hasn't been completely removed yet, I can see at least 5 references.